### ISSUE TITLE
Release build is broken (1.6 / Sitecore 9.0 Update-2)

### ISSUE THREAD

**Issue by nickwesselman** [Sitecore/Habitat#435]
**Describe the bug**
Habitat cannot currently build in the Release configuration:
```
src\Foundation\Dictionary\tests\Services\CreateDictionaryEntryServiceTests.cs(29,30): Error CS0122: 'CreateDictionaryEntryService' is inaccessible due to its protection level
```

**Sitecore Version**
Sitecore 9.0 Update-2

**To Reproduce**
1. Clone repo
2. Switch to Release build
3. Build

**Expected behavior**
Successful build

**Additional context**
`CreateDictionaryEntryServiceTests` is testing the `internal` marked `CreateDictionaryEntryService`. However the `InternalsVisibleTo` assembly property is conditional to `DEBUG` only.

**Comment by BenMurphy1994** [comment]
Any update on this bug?

**Comment by nickwesselman** [comment]
Hi @BenMurphy1994 -- Release build not typically needed for Habitat, as it is an example reference intended for deployment in local development environments, so this is not currently a high priority. It will likely be addressed when Habitat is released for the next major release of Sitecore itself. Is there a specific use case that this is blocking for you?

It can easily be worked around by disabling compilation of `Foundation.Dictionary.Tests` in the `Release` build configuration, or removing the conditional compilation on the `InternalsVisibleTo`.

**Comment by BenMurphy1994** [comment]
Nothing major, just looking to demo an Azure Sitecore site to a client. Using habitat as the demo site and releasing through Azure DevOps as a personal training objective. 

Okay thank you, will remove from the Release build config. Thanks for the reply!

**Comment by nickwesselman** [comment]
Sounds good. For demo purposes you should just be able to use a Debug build as well I would think.

**Comment by nickwesselman** [comment]
Resolved in Habitat 1.7
